website/blog/2023-01-18-release-0.11.md

```diff
@@ -0,0 +1,256 @@
+
+In case the compatibility mode is not activated, new documentation on how to activate it and how to easily switch between Docker and Podman is available here: [https://podman-desktop.io/docs/migrating-from-docker](https://podman-desktop.io/docs/migrating-from-docker). 
+
+This is a warning displayed for Windows users:
+![windows-docker-compatibility](https://user-images.githubusercontent.com/436777/209305744-9056addd-f122-4f75-9e03-ad27a0109375.png)
+
+This is the warning for Mac users:
+![mac-docker-compatibility](https://user-images.githubusercontent.com/6422176/207964861-a9c1f72c-89d6-4816-beab-397af4125620.png)
+
+
+### Allow to toggle on/off the proxy setting [#983](https://github.com/containers/podman-desktop/pull/983)
+
+In some context, users need the ability to disable and re-enable the proxy configuration very quickly, without havint to entirely reconfigure it. This is now possible from the Podman Desktop settings page, where a toggle to enable/disable the proxy configuration has been added.
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/205955418-670bc37c-a74f-40ef-bc60-8d9d013aa0dc.mp4' />
+
+Note: extensions can read this information and then update the proxy configuration.
+
+### Namespace selection for deploy to kubernetes [#1008](https://github.com/containers/podman-desktop/pull/1008) 
+
+A new dropdown menu has been added to the deploy to Kubernetes screen to allow selecting the namespace to deploy to. Defaults to defaultnamespace, if unable to get any from kube_client module.
+
+If the user have set `kubectl config set-context --current --namespace=NAMESPACE` it honors that as the chosen value.
+
+![deploy-kubernetes-namespace](https://user-images.githubusercontent.com/7339809/206688886-095e4f15-42ae-4a0a-b1c6-ae4b547fcdfb.gif)
+
+### Configure Podman binary path [#941](https://github.com/containers/podman-desktop/pull/941)
+
+Users may use a custom path to the podman binary. An option within settings (Settings -> Preferences) to allow providing an additional path to Podman binary has been added. This is useful for users who have installed Podman using a package manager and want to use the binary provided by the package manager. Additional information is available in the documentation for Mac users: [http://podman-desktop.io/docs/troubleshooting#unable-to-set-custom-binary-path-for-podman-on-macos](http://podman-desktop.io/docs/troubleshooting#unable-to-set-custom-binary-path-for-podman-on-macos)
+  
+![custom-path](https://user-images.githubusercontent.com/6422176/204832063-5858065a-2fc3-49de-8d23-3b99b7d10dbf.png)
+
+### Provide pre-defined registries [#1201](https://github.com/containers/podman-desktop/pull/1201)
+
+The experience of configuring a registry is getting simplified for the most popular ones. The user will be able to select a registry from a pre-defined list and will only have to provide the username and password. The list of pre-defined registries is the following:
```

Comment:
"The list of pre-defined registries is the following:" -> "The following registries will be pre-defined:"?